lib/zendesk_apps_tools/command.rb

```diff
@@ -150,5 +191,57 @@ def setup_path(path)
       @destination_stack << relative_to_original_destination_root(path) unless @destination_stack.last == path
     end
 
+    def config_dir
+      @config_dir ||= begin
+        if OS.mac?
+          File.join(Dir.home, 'Library/Application Support/ZAT')
+        elsif OS.windows?
+          File.join(ENV['APPDATA'] || Dir.home, 'ZAT')
+        elsif ENV['XDG_CONFIG_HOME']
+          File.join(ENV['XDG_CONFIG_HOME'], 'zat')
+        else
+          File.join(Dir.home, '.zat')
+        end
+      end
+      Dir.mkdir @config_dir unless Dir.exists? @config_dir
+      @config_dir
+    end
+
+    def localhost_ssl_cert_path
+      File.join(config_dir, 'localhost.pem')
+    end
+
+    def localhost_ssl_key_path
+      File.join(config_dir, 'localhost.key')
+    end
+
+    def generate_certificate
+      puts "Generating certificate..."
+      certificate = SelfSignedCertificate.new
+      File.write(localhost_ssl_cert_path, certificate.self_signed_pem)
+      File.write(localhost_ssl_key_path, certificate.private_key)
+    end
+
+    def trust_certificate
+      puts "Trusting certificate..."
+      if OS.mac?
+        exec_cmd("sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain \"#{localhost_ssl_cert_path}\"")
+      elsif OS.windows?
+        exec_cmd("certutil.exe -addstore -user root \"#{localhost_ssl_cert_path}\"")
```

Comment:
Manual or drop Windows support (of this functionality) for now. I'm pretty sure not many Ruby developers use Windows. From my own experience, working on Ruby on Windows is quite painful.